src/transformed_distribution.jl

```diff
@@ -101,8 +100,8 @@ end
 ##############################
 
 # size
-Base.length(td::Transformed) = length(td.dist)
-Base.size(td::Transformed) = size(td.dist)
+Base.length(td::Transformed) = output_length(td.transform, size(td.dist))
```

Comment:
It seems this is the only time `output_length` is used:
```suggestion
Base.length(td::Transformed) = only(output_size(td.transform, size(td.dist)))